src/d3a/constants.py

```diff
@@ -44,3 +45,8 @@
 CN_PROFILE_EXPANSION_DAYS = 7
 
 RUN_IN_REALTIME = False
+
+
+class BidOfferMatchEnum(Enum):
```

Comment:
As there is already a `BidOfferMatch` object that holds the actual match, this naming here could lead to confusion (just experienced it ;-) ) 
I suggest to rename to: BidOfferMatchAlgoEnum